add tests for osn transition

summeroff opened the pull request.

summeroff committed:
test: add tests for osn transition

pdamasceno commented on tests/osn-tests/src/test_osn_transition.ts:
```diff
@@ -0,0 +1,150 @@
+
+    });
+
+    context('# createPrivate', () => {
```
The frontend don't use CreatePrivate. I added this function to the list of functions not being used. I think it is best to remove this test case until we decide what to do with this function in the backend

pdamasceno commented on tests/osn-tests/src/test_osn_transition.ts:
```diff
@@ -0,0 +1,150 @@
+       
+    });
+
+    context('# set', () => {
```
Add getActiveSource and clear to the description of context here to inform that those functions are also being tested here

pdamasceno reviewed (changes requested):
Minor changes that I proposed. Everything is great with the test cases of osn-transition.

pdamasceno commented on tests/osn-tests/src/test_osn_transition.ts:
```diff
@@ -0,0 +1,150 @@
+                // Checking if transition was created correctly
+                expect(transition).to.not.equal(undefined);
+                expect(transition.id).to.equal(transitionType);
+                expect(transition.name).to.equal(transitionName);
```
Verify if the setting you created the transition with is present in the list of settings. See how it is done in osn-input test case.

summeroff committed:
remove test for unused function

update description to include all tested functions
add check for test with settings
change how exception checked to reflect frontend use
